ci/e2e-tests.sh

```diff
@@ -68,6 +68,14 @@ get_package() {
             artifact_name='debian-10-slim'
             ;;
 
+        'debian:11')
+            artifact_name='debian-11-slim'
+            ;;
+
+        'platform:el8')
+            artifact_name='el-8'
```

Comment:
This is wrong. Needs to be `redhat-ubi8-latest`. I'm fixing it on master as part of fixing E2E tests, so wait for that.